src/main/kotlin/inc/evil/d17map/security/jwt/JWTService.kt

```diff
@@ -0,0 +1,65 @@
+package inc.evil.d17map.security.jwt
+
+
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jwts
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.stereotype.Service
+import java.security.NoSuchAlgorithmException
+import java.util.*
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+
+@Service
+class JWTService {
+
+    private val sec: SecretKey = try {
+        val keyGenerator = KeyGenerator.getInstance("HmacSHA256")
```

Comment:
Tutaj też chyba dobrze byłoby to przypisać do jakiejś stałej

````
companion object {
    private const val ALGORITHM = "HmacSHA256"
}


A najlepiej chyba byłoby to zdefiniować jako zmienna środowiskowa, żeby nie przechowywać tego raczej nigdzie w kodzie.

Reply:
A najlepiej chyba byłoby to zdefiniować jako zmienna środowiskowa, żeby nie przechowywać tego raczej nigdzie w kodzie.

Reply:
Nie jestem pewny czy użyty algorytm jest aż tak wrażliwą daną? `RSA` jest stosowany w wielu systemach a nadal pozostaje bezpieczny...

Reply:
Dałem tego stringa do properties, ale w kodzie jest defaultowa wartosć `HmacSHA256`